### ISSUE TITLE
Issue Downloading ElectronCash-3.1.5.exe

### ISSUE THREAD

**Issue by powe11tm** [fyookball/electrum#601]
When trying to download ElectronCash-3.1.5.exe on my PC with Windows 10, it fails to download and is blocked.  I get a message saying that ElectronCash-3.1.5 is not safe and was blocked by Windows Defender Smart Screen.  Unsure how to proceed.  I'm at what appears to be the correct site, https://www.electroncash.org/ and went to the site from the link on https://www.electroncash.org/ so I'm not sure how to proceed.  If I turn off Windows Defender, I don't want to download an unsafe file.  Is this a known issue, and what do you recommend?  Thanks.

**Comment by powe11tm** [comment]
The link to the site I used was posted on https://www.bitcoincash.org

**Comment by fyookball** [comment]
just double check the checksum before you run it....and add an exception to your windows defender for that file.

**Comment by powe11tm** [comment]
So I disabled Windows Defender, and then my anti-virus software (Norton) flagged the file as an unsafe file.  It won't verify the signatures or publisher, so it's blocked again.  Also, I tried open the checksum file under sigs-and-sums, 3.1.5, then win-Linux, since I have Windows 10.  That file won't open properly and it gives me a bunch of funny characters.  Not sure if it is opening with the right program.  I'm no IT guy, and this seems overly complicated for your average person.  It would work much better for mass adoption if this downloaded easily without getting blocked with verified signatures from a verified source.  I'd like to try it, but so far I'm not having any luck.
